client-grpc/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+edition = "2021"
+name    = "svc-cargo-client-grpc"
+version = "0.6.0-develop.1"
```

Comment:
Please reset the version to "0.1.0" so the release workflow won't run into issues and we get a proper CHANGELOG generated

Reply:
https://github.com/Arrow-air/svc-cargo/pull/16/commits/bbc5a4b421abcb5923ad3701a58f8af7c5e682b4